src/Server.ts

```diff
@@ -0,0 +1,85 @@
+import assert from 'assert'
+import express, { Express } from 'express'
+import * as http from 'http'
+
+import Logger from 'helpers/Logger'
+import { Request, ParamsDictionary, Response, NextFunction } from 'express-serve-static-core'
+
+const log = new Logger('server')
+export interface Params {
+  port: number
+}
+
+export class Server {
+  private _port: number
+  private _server: http.Server | null = null
+
+  constructor (params: Params) {
+    const { port } = params
+    this._port = port
+  }
+
+  public start (): Promise<void> {
+    log.debug(`Starting server on port ${this._port}...`)
+    assert(this._server === null, 'Server was already started')
+    const app = express()
+
+    // Register endpoints
+    this._registerEndpoint(app)
+
+    // Register middleware
+    this._registerMiddleware(app)
+
+    // Create server
+    this._server = http.createServer(app)
+    const server = this._server as http.Server
+
+    // Error handling
+    server.on('error', log.errorHandler)
+
+    // Start to listen on port
+    return new Promise((resolve, reject) => {
+      server.once('error', reject)
+      server.listen(this._port, () => {
+        log.debug(`Server started on port ${this._port}!`)
+        log.debug(`Ping URL: http://localhost:${this._port}/v1/health/ping`)
+        log.debug(`Alive URL: http://localhost:${this._port}/v1/health/healthy`)
+        resolve()
+      })
+    })
+  }
+
+  private _registerEndpoint (app: Express) {
+    app.get('/v1/health/ping', (_req, res) => res.status(204).send())
+    app.get('/v1/health/healthy', (_req, res) => res.status(204).send())
```

Comment:
As 204 response is cacheable by default, it may be a good idea to do
```
res.set('Cache-Control', 'no-store, no-cache, must-revalidate, private')
```

Reply:
![image](https://user-images.githubusercontent.com/2352112/70225255-f72c8780-174e-11ea-9a4d-89967c5eecf6.png)

Also added another one, with the version, with a cache of 2min (will implement in next PR):
![image](https://user-images.githubusercontent.com/2352112/70225335-1cb99100-174f-11ea-886a-e50f6434dd14.png)